pkgs/development/compilers/llvm/README.md

```diff
@@ -64,6 +64,21 @@
 
 ## Information on our current patch sets
 
+### Adding a patch
+
+To add an LLVM patch in the Nixpkgs tree,
+
+1. Put the patch in the corresponding directory (`<VERSION>/<PACKAGE>`).
+   > If you want your patch to apply to clang version 12 (and, optionally, later versions), put it in `./12/clang`.
```

Comment:
`>` is quote syntax. These are not quotes.

Reply:
Yeah, they started out as `[!TIP]` but those don't work in lists. I'll reformat.